eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/SiteAccessAware/Contextualizer.php

```diff
@@ -298,4 +298,21 @@ public function getGroupsBySiteAccess()
     {
         return $this->groupsBySiteAccess;
     }
+
+    private function array_merge_unshift(...$args)
```

Comment:
A little PHPDoc would be nice here, as code is hard to understand (by design, not your fault) 😃